src/SFA.DAS.AANHub.Api.UnitTests/Controllers/WhenPostingCreateMember.cs

```diff
@@ -23,18 +23,6 @@ public WhenPostingCreateMember()
             _controller = new MemberController(_mediator.Object, Mock.Of<ILogger<MemberController>>());
         }
 
-        [Test, AutoMoqData]
```

Comment:
Why is this being removed?

Reply:
This is an outdated endpoint test, and probably should have been removed as part of PR #26.

The work in that PR has superceded this.